cmd/ci-secret-bootstrap/main.go

```diff
@@ -672,23 +672,63 @@ func getUnusedBWItems(config secretbootstrap.Config, client secrets.Client, bwAl
 	return utilerrors.NewAggregate(errs)
 }
 
+func validateBWItems(client secrets.Client, secretConfigs []secretbootstrap.SecretConfig) error {
+	var errs []error
+
+	for _, config := range secretConfigs {
+		for _, item := range config.From {
+
+			if item.DockerConfigJSONData != nil {
+				for _, data := range item.DockerConfigJSONData {
+					if !client.HasItem(data.BWItem) {
+						errs = append(errs, fmt.Errorf("item %s doesn't exist", data.BWItem))
+						continue
+					}
+					if _, err := client.GetAttachmentOnItem(data.BWItem, data.AuthBitwardenAttachment); err != nil {
+						errs = append(errs, fmt.Errorf("attachment %s in item %s doesn't exist", data.AuthBitwardenAttachment, data.BWItem))
+						continue
+					}
+				}
+			} else {
+				if !client.HasItem(item.BWItem) {
+					errs = append(errs, fmt.Errorf("item %s doesn't exist", item.BWItem))
+					continue
+				}
+
+				if item.Field != "" {
+					if _, err := client.GetFieldOnItem(item.BWItem, item.Field); err != nil {
+						errs = append(errs, fmt.Errorf("field %s in item %s doesn't exist", item.Field, item.BWItem))
+						continue
```

Comment:
These `continue`s` (this one and the ones below) do not make sense to me. If we expect exactly one of the `if`s to apply, then the three `if` statements should be a switch, or we expect multiple ones to apply, and then we should not abort on finding a first error.

Reply:
Nice catch. I will update it. Thanks

Reply:
updated